code/modules/mob/living/simple_animal/hostile/true_changeling.dm

```diff
@@ -0,0 +1,367 @@
+	see_in_dark = 8
+	see_invisible = SEE_INVISIBLE_MINIMUM
+	wander = 0
+	attacktext = "tears into"
+	attack_sound = 'sound/creatures/hit3.ogg'
+	butcher_results = list(/obj/item/reagent_containers/food/snacks/meat/slab = 15) //It's a pretty big dude. Actually killing one is a feat.
+	var/playstyle_string = "<b><span class='big danger'>We have entered our true form!</span><br>We are unbelievably deadly, and regenerate life at a steady rate. We must utilise the abilities that we have gained as a result of our transformation, as our old ones are not usable in this form. Taking too much damage will also turn us back into a \
+	human in addition to knocking us out. We are not as strong health-wise as we are damage, and we must avoid fire at all costs. Finally, we will uncontrollably revert into a human after some time due to our inability to maintain this form.</b>"
+	var/mob/living/carbon/human/stored_changeling = null //The changeling that transformed
+
+	var/devouring = FALSE
+	var/assimilation = FALSE
+	var/tendrilgrab = FALSE
+	var/wallcrawl = FALSE
+
+	var/range = 7
+	var/datum/action/innate/changeling/devour/devour
+	var/datum/action/innate/changeling/tendril_grab/tendril_grab
+	var/datum/action/innate/changeling/assimilate/assimilate
+	var/datum/action/innate/changeling/spine_crawl/spine_crawl
+
+/mob/living/simple_animal/hostile/true_changeling/Initialize()
+	. = ..()
+	icon_state = "horror[rand(1, 5)]"
+	devour = new
+	devour.Grant(src)
+	tendril_grab = new
+	tendril_grab.Grant(src)
+	assimilate = new
+	assimilate.Grant(src)
+	spine_crawl = new
+	spine_crawl.Grant(src)
+	var/datum/atom_hud/hud = GLOB.huds[ANTAG_HUD_CHANGELING]
+	hud.add_hud_to(src)
+
+/mob/living/simple_animal/hostile/true_changeling/Destroy()
+	QDEL_NULL(devour)
+	QDEL_NULL(tendril_grab)
+	QDEL_NULL(assimilate)
+	QDEL_NULL(spine_crawl)
+	stored_changeling = null
+	var/datum/atom_hud/antag/hud = GLOB.huds[ANTAG_HUD_CHANGELING]
+	hud.remove_hud_from(src)
+	return ..()
+
+/mob/living/simple_animal/hostile/true_changeling/Login()
+	. = ..()
+	to_chat(usr, playstyle_string)
+
+/mob/living/simple_animal/hostile/true_changeling/Life()
+	..()
+	adjustBruteLoss(-TRUE_CHANGELING_PASSIVE_HEAL) //True changelings slowly regenerate
+
+/mob/living/simple_animal/hostile/true_changeling/Stat()
+	..()
+	if(statpanel("Status"))
+		stat(null, "Ignoring Gravity: [wallcrawl ? "YES" : "NO"]")
+
+/mob/living/simple_animal/hostile/true_changeling/death()
+	..(1)
+	new /obj/effect/gibspawner/human(get_turf(src))
+	if(stored_changeling && mind)
+		visible_message("<span class='warning'>[src] lets out a furious scream as it shrinks into its human form.</span>", \
+						"<span class='userdanger'>We lack the power to maintain this form! We helplessly turn back into a human...</span>")
+		stored_changeling.loc = get_turf(src)
+		mind.transfer_to(stored_changeling)
+		stored_changeling.Unconscious(300) //Make them helpless for some time
+		stored_changeling.status_flags &= ~GODMODE
```

Comment:
Move this up a line or the unconscious ling would not be exactly 'helpless'.

Reply:
It's not a sleep don't worry BUTT

I should really gut the code anyways, once the ling goes horror it should not go back to human

Reply:
Right, silly me.